Admin/Extension/MenuNodeReferrersExtension.php

```diff
@@ -21,20 +21,47 @@
  */
 class MenuNodeReferrersExtension extends AdminExtension
 {
+    /**
+     * @var string
+     */
+    private $formTab;
+
+    /**
+     * @var string
+     */
+    private $formGroup;
+
+    public function __construct($formGroup = 'form.group_menu', $formTab = 'form.tab_menu')
+    {
+        $this->formGroup = $formGroup;
+        $this->formTab = $formTab;
+    }
+
     public function configureFormFields(FormMapper $formMapper)
     {
+        if ($formMapper->hasOpenTab()) {
+            $formMapper->end();
+        }
```

Comment:
@dbu this if statement requires SonataAdminBundle 2.4 (the `hasOpenTab()` method is introduced in 2.4 only https://github.com/sonata-project/SonataAdminBundle/pull/2680, the `->tab()` methods can be replaced with `->with(..., ['tab' => true])` for SonataAdminBundle 2.3 support).

Looking at [`->end()`](https://github.com/WouterJ/SonataAdminBundle/blob/34648f52f2a7f2c8d018779b00e535387da4c6ee/Mapper/BaseGroupedMapper.php#L210-L221), I think we can also just call the method and catch the RuntimeException in case no tab is currently open. While it is not really nice, I think this would allow us to support SonataAdminBundle 2.3.

Btw, if we are going to include this in the 2.0 release, I think we should also update all other tab PRs and merge them into 1.3/1.4 for consistency in the admin gui.